modules/TodoAppPage/containers/TodoAppContainer.tsx

```diff
@@ -1,22 +1,22 @@
 import { useEffect, useState } from "react";
 import { Flex } from "@mantine/core";
-import MenuBar from "./Pages/HomePage/components/MenuBar/MenuBar";
 import { useDisclosure } from "@mantine/hooks";
 import dayjs from "dayjs";
 
-import CreateTask from "./Pages/HomePage/components/CreateTask/CreateTask";
-import DisplayTask from "./Pages/HomePage/components/DisplayTask/DisplayTask";
-import { Task } from "./types/Task";
+import { TTask } from "@/shared/typedefs/types";
+import CreateTask from "../components/CreateTask/CreateTask";
+import DisplayTask from "../components/DisplayTask/DisplayTask";
+import MenuBar from "../components/MenuBar/MenuBar";
 import {
   setTasksAtLocalStorage,
   deleteCompletedTasksFromLocalStorage,
   getSortedTasks,
-} from "./Shared/Utils/localstorage";
+} from "../../../shared/utils/localStorage";
```

Comment:
```suggestion
} from "@/shared/utils/localStorage";
```

This may be an @ import